themes/navy/layout/partial/after_footer.swig

```diff
@@ -15,6 +16,28 @@
 {{ js('js/plugins') }}
 <!-- endbuild -->
 {% endif %}
+<script async src="/js/highlight.pack.js"></script>
+<script async src="/js/solidity.js"></script>
+<script>
+
+function highlight() {
+  $('pre code').each(function(i, block) {
+    hljs.highlightBlock(block);
+  });
+}
+
+$(document).ready(function() {
+    try {
+      highlight();
+    } catch(err) {
```

Comment:
Why would it fail?

Reply:
it fails sometimes, seems to be a known issue, document ready is suppose to prevent it but still